imageio/plugins/_freeimage.py

```diff
@@ -399,7 +400,30 @@ def error_handler(fif, message):
         
         # Make sure to keep a ref to function
         self._error_handler = error_handler
-        
+    
+    @property
```

Comment:
Yes, we could generalize this, but since freeimage and avbin are the only plugins that use a library, I think we can hold that off until we get more such plugins.